releases/codefresh.yaml

```diff
@@ -0,0 +1,259 @@
+repositories:
+# Codefresh on-prem repository
+- name: "codefresh"
+  url: 'http://charts.codefresh.io/{{ env "CHANNEL" | default "dev" }}'
+# Incubator repo of official helm charts
+- name: "incubator"
+  url: "https://kubernetes-charts-incubator.storage.googleapis.com/"
+
+releases:
+
+#######################################################################################
+## Codefresh On-Prem                                                                 ##
+#######################################################################################
+
+#
+# References:
+#   - https://github.com/codefresh-io/onprem
+#
+- name: "cf"
```

Comment:
should the `name` be more descriptive (maybe `codefresh-onprem` or `codefresh`)?
I understand the shorter the name the better, but when I don't know what was deployed on the cluster, and do `helm list` or `kubectl get pods`, the name `cf` does not convey what it is

Reply:
no. we use naming that codefresh provides, to simplify support in the future.